Icon/Icon+PreviewHelpers.swift

```diff
@@ -5,6 +5,8 @@
 //  Copyright © 2020 Velos Mobile LLC / https://velosmobile.com / All rights reserved.
 //
 
+// swiftlint:disable line_length
```

Comment:
☝️ Do you mind removing these swiftlint disables from the PR?

I totally get why you've added them since the current instructions have you copying these files into your project directly and your local swiftlint is throwing some warnings, but I think when we can get #2 merged, you'd be able to remove the files directly from the project.